src/geocat/viz/util.py

```diff
@@ -1279,34 +1476,52 @@ def get_skewt_vars(p: Quantity, tc: Quantity, tdc: Quantity,
 
     - `NCL_skewt_2_2 <https://geocat-examples.readthedocs.io/en/latest/gallery/Skew-T/NCL_skewt_2_2.html?highlight=get_skewt_vars>`_
     """
+    # Support for deprecating kwargs
```

Comment:
Does this seem like a good solution @kafitzgerald ? I've never done this before. 
@anissa111 has geocat-comp ever done something like this?

I can see how it would be more robust by adding checks to make sure that pressure and temperature etc exist if not listed (eventually we don't want them to be optional but it needs to be for now to support either choice).

Reply:
I have also never done a deprecated kwarg before. I don't have any problem with how you've done it here, but I would recommend checking how other packages have handled this. Maybe PEP has a standardized recommendation?

Reply:
I don't have a solid recommendation offhand and it doesn't look like PEP has a lot of guidance ([here's the relevant one for backwards compatibility](https://peps.python.org/pep-0387/)), but there is some info on usage of `DeprecationWarning` and `PendingDeprecationWarning`.  And Xarray and Matplotlib should have some recent examples I think.